docs/topics/kotlin-evolution.md

```diff
@@ -205,4 +205,21 @@ For fully stable versions of the compiler, the default binary compatibility prot
 
 This protocol is designed for comfortable updates as no project can be blocked from updating its dependencies even if it's using a slightly outdated compiler.
 
-Please note that not all target platforms have reached this level of stability (but Kotlin/JVM has).
\ No newline at end of file
+Note that not all target platforms have reached this level of stability, but Kotlin/JVM has.
+
+#### Kotlin klib binaries
+
+Kotlin klib binaries have reached the [Stable](components-stability.md#stability-levels-explained) level in Kotlin 1.9.20.
+However, there are some compatibility details you need to keep in mind:
+
+* klib binaries are backwards compatible starting with Kotlin 1.9.20. For example, the 2.0.* compiler can read binaries
+  produced by the 1.9.* compiler.
+* klib binaries are compatible within the same language or feature version. For example, the 2.0.* compiler can read
```

Comment:
I would probably use the same wording as in the previous section (for JVM) to avoid misunderstanding that this rule is different for KLIBs and JARs. Which is: `Older compilers reject klibs that rely on new features (e.g. a 1.9.20 compiler rejects binaries that use 2.0.*-specific language features)`

Reply:
Thank you! Added a line about older compilers

Reply:
I meant that the whole line can be replaced by the line that you've actually added.

Reply:
Ah, I see. thank you for the clarification!